src/components/VideoCapture/index.tsx

```diff
@@ -36,7 +36,7 @@ export type Props = {
   cameraClassName?: string
   facing?: VideoFacingModeEnum
   inactiveError: ErrorProp
-  method: CaptureMethods
+  method: Exclude<CaptureMethods, 'poa'>
```

Comment:
Make it an own type?

Reply:
I'll make sure to include this suggestion in the work I'm doing in a related issue.